django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,40 @@
+import uuid
```

Comment:
Is this import needed anymore, given the change to ulid?

Reply:
Oops, this is flagged by flake8 but since we can't make it mandatory yet it got past